tests/wasm/mod.rs

```diff
@@ -15,7 +15,6 @@ use process_control::Output;
 use rustls::client::{ServerCertVerified, ServerCertVerifier};
 use rustls::version::TLS13;
 use rustls::Certificate;
-use serial_test::serial;
```

Comment:
Why? Doesn't this then overload on SEV-SNP, which was the reason why we introduced this.

Reply:
So far it looks like it works fine.

Without this, tests consistently time out and take forever locally.
Tests are only useful if they are simple to run and run fast.

Is there a config I can use, which would only enable this on SEV?

Reply:
ok, fine for now... we can reintroduce it, if it causes unexpected failures.